lib/modules/resumemodule/resumemodule.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict
+
+import boto3
+import requests
+import io
+from botocore.exceptions import ClientError
+from lib.modules.modulehelpers import *
+from lib.modules.moduletemplate import ModuleTemplate
+from lib.modules.resumemodule.resumeresponse import *
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+import PyPDF2
+
+env_path = Path('.') / '.env'
+load_dotenv(dotenv_path=env_path)
+
+
+s3 = boto3.client('s3')
+
+class ResumeModule(ModuleTemplate):
+
+    def interpret_message(self, msg, email, db, admin) -> Dict:
+        commands = msg["text"].split()
+
+        if commands[0] == 'resume':
+            commands = commands[1:]
+            if len(commands) == 0:
+                return
+
+            if commands[0] == 'list_resumes':
+                if len(commands) > 2:
+                    return catch_incorrect_arguments(Commands.LIST_RESUMES)
+                
+                return {
+                    "command": Commands.LIST_RESUMES,
+                    "id": commands[1] if len(commands) == 2 else None
+                }
+            
+            if commands[0] == 'add_resume':
+                if len(commands) != 1:
+                    return catch_incorrect_arguments(Commands.ADD_RESUME)
+                return {
+                    "command": Commands.ADD_RESUME
+                }
+            
+            if commands[0] == 'remove_resume':
+                if len(commands) != 1:
+                    return catch_incorrect_arguments(Commands.REMOVE_RESUME)
+                return {
+                    "command": Commands.REMOVE_RESUME
+                }
+
+    def process_message(self, interpretation, client, req, email, db: database, admin) -> Dict:
```

Comment:
There is complicated logic here, could you add some docs for this function explaining in English how it works logically